js/constants/constants.js

```diff
@@ -0,0 +1,8 @@
+var keyMirror = require('react/lib/keyMirror');
+
+module.exports = {
+  ActionTypes: keyMirror({
+    NEW_MESSAGE: null,
+    RENAME_LOCAL_USER: null
+  })
```

Comment:
good question. in the meantime, we may have to write a matcher that checks for equality AND not null :disappointed: